lib/params.js

```diff
@@ -1,44 +1,51 @@
-var fs = require('fs')
-var path = require('path')
+const fs = require('fs')
+const path = require('path')
+const createStylelint = require('stylelint').createLinter
+const editorconfig = require('editorconfig')
+const getProperty = require('./util').getProperty
+const defaultIndentWidth = ' '.repeat(2)
+const stylelintCache = new Map()
```

Comment:
Why cache? Is it assumed to be executed repeatedly in e.g. gulp? I think, it is not necessary because we use `params.js` in `index.js`.